subprojects/shipkit/src/main/groovy/org/shipkit/gradle/configuration/ShipkitConfiguration.java

```diff
@@ -360,6 +360,34 @@ public void setCommitMessagePostfix(String commitMessagePostfix) {
             //TODO protect this setter and other relevant from invalid input (null value)
             store.put("git.commitMessagePostfix", commitMessagePostfix);
         }
+
+        /**
+         * Text will be used as description for pull request to downstream repositories.
+         */
+        public String getPullRequestDescription() {
+            return store.getString("git.pullRequestDescription");
```

Comment:
We don't need properties in ShipkitConfiguration at the moment. We don't want it to grow more than necessary. You can start with only task properties, so that you could modify task like that:
project.tasks.createPullRequest.pullRequestDescription = "Upgraded version"